src/openrct2-ui/input/MouseInput.cpp

```diff
@@ -705,8 +706,7 @@ static void input_scroll_continue(rct_window* w, rct_widgetindex widgetIndex, Sc
         return;
     }
 
-    screenCoords.x = x2;
-    screenCoords.y = y2;
+    screenCoords = newScreenCoords;
```

Comment:
This doesn't need to be done. The scroll coords can be passed directly to the mouse drag call

Reply:
Done